runtime/tests/jvmtitests/agent/version.c

```diff
@@ -30,7 +30,7 @@ const static char *versionNames[] =
 	"JVMTI 1.2",
 	"JVMTI 9.0",
 	"JVMTI 11",
-	"JVMTI 15",
+	"JVMTI " JAVA_SPEC_VERSION_STRING,
```

Comment:
This doesn't seem like it will work in the future - when we support JVMTI 50, do we not still want to allow people to ask for JVMTI 15? Similar comment below for the GetEnv version check.

Reply:
The goal is to keep the support of `JVMTI_VERSION_15` since it was in `jvmti.h` already. 
The code commented is OpenJ9 test though, I think it is free for modification. Similarly for `getVersionName()` in the same test.

Reply:
Is `JVMTI_VERSION` essentially jvmti_latest? Meaning that at some point we will add `JVMTI 17` ?

Reply:
Correct.

From `JDK16` `jvmti.h`
```
JVMTI_VERSION = 0x30000000 + (16 * 0x10000) + ( 0 * 0x100) + 0  /* version: 16.0.0 */
```

From `JDK17` `jvmti.h`
```
JVMTI_VERSION = 0x30000000 + (17 * 0x10000) + ( 0 * 0x100) + 0  /* version: 17.0.0 */
```